Quadriga/src/main/java/edu/asu/spring/quadriga/mapper/PassThroughProjectDTOMapper.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.spring.quadriga.mapper;
+
+import java.util.Date;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.IUser;
+import edu.asu.spring.quadriga.domain.passthroughproject.IPassThroughProject;
+import edu.asu.spring.quadriga.dto.PassThroughProjectDTO;
+
+/**
+ * The purpose of this class is to map the PassThroughProjectDTO class objects
+ * to the PassThroughProject domain objects used in Quadriga
+ * 
+ */
+@Service
+public class PassThroughProjectDTOMapper extends BaseMapper {
```

Comment:
if this class inherits from the ProjectDTOmapper you can override the mapping method and don't have to recreate all the base project dto setter calls.